frontend/src/components/task.jsx

```diff
@@ -1,7 +1,7 @@
 import styles from "./task.module.css"
 
 export default function Task(props) {
-    const { task, setTasks, update, setUpdate } = props;
+    const { task, setTasks } = props;
```

Comment:
We're no longer using the `update` and `setUpdate` state variables as explained [in this comment](https://github.com/nds-fsd/Abd-ha-todo-app/pull/3/files#r1505501386).